### ISSUE TITLE
[LFH]: "Instant sell NFT" marketplace aggregator → need front end

### ISSUE THREAD

**Issue by theode69** [ethb3rlin/find-a-team#55]
## Pitch it!

TLDR: Instantly sell your NFT at the best price of the market, in a click. 

So far, all marketplaces and aggregators have been designed to maximize the "buyer" experience. 
We propose to flip the paradigm and build an aggregator designed for sellers. 

Use our app and instant sell your illiquid jpeg at the best market price, unlocking liquidity for holders 

## Team

_Talk a little about the team:_

- [x] [theo](https://twitter.com/theo__0101) : product
- [x] [mick](https://twitter.com/MickdeG010) : sc, backend 

## Skills Wanted

- [x] DApps, Web3.JS (JavaScript, CSS)
- We are looking for a full stack/front-end dev. Appreciate if you have xp with web3 stack 

## Communication

- Discord: theo__#9243
- Matrix: @theode69:matrix.org
